receiver/opencensusreceiver/config.go

```diff
@@ -48,18 +43,6 @@ type Config struct {
 	MaxConcurrentStreams uint32 `mapstructure:"max-concurrent-streams,omitempty"`
 }
 
-// tlsCredentials holds the fields for TLS credentials
-// that are used for starting a server.
-// TODO(ccaraman): Add validation to check that these files exist at configuration loading time.
```

Comment:
I think I should address this comment by @ccaraman in this PR, but I'm not sure where the best place to validate this for all receivers would be.